hazelcast/src/main/java/com/hazelcast/sql/SqlColumnType.java

```diff
@@ -67,7 +67,14 @@
     TIMESTAMP_WITH_TIME_ZONE(OffsetDateTime.class),
 
     /** OBJECT type, could be represented by any Java class. */
-    OBJECT(Object.class);
+    OBJECT(Object.class),
```

Comment:
`INT` type is displayed as `INTEGER` in Calcite exception. I think we'd better rename our type to `INTEGER`

Reply:
fixed int 6609ac14ce29d55f387ba2bc830b537120e1d86f